lib/matplotlib/lines.py

```diff
@@ -790,6 +790,9 @@ def draw(self, renderer):
                     cap = self._dashcapstyle
                     join = self._dashjoinstyle
                 else:
+                    print(tpath.vertices.size)
+                    tpath = affine.inverted().transform_path(affine.transform_path(tpath).cleaned(simplify=True))
+                    print(tpath.vertices.size)
```

Comment:
You may be able to get away with editing _classic_test.mplstyle, which is basically there for that reason (we try not to update all test images at once if possible, as this really leads to bloating of the repo size).